src/tsickle.ts

```diff
@@ -908,12 +908,12 @@ class ExternsWriter extends ClosureRewriter {
           break;
         case ts.SyntaxKind.MethodSignature:
         case ts.SyntaxKind.MethodDeclaration:
-          let method = <ts.MethodDeclaration>member;
-          let overloaded = methods[method.name.getText()];
-          if (overloaded) {
-            methods[method.name.getText()].push(method);
+          const method = member as ts.MethodDeclaration;
+          const methodName = method.name.getText();
+          if (methods.has(methodName)) {
+            methods.get(methodName)!.push(method);
```

Comment:
I wish they added map.setdefault method, like in python, then you can do `map.setDefault(key, []).push(...)`. 

no change needed.